bitnami/grafana-operator/templates/grafana.yaml

```diff
@@ -118,6 +118,9 @@ spec:
       {{- end }}
     {{- end }}
     {{- if .Values.grafana.updateStrategy }}
+    nodeSelector: {{ toYaml .Values.grafana.nodeSelector | nindent 6 }}
+    {{- end }}
+    {{- if .Values.grafana.updateStrategy }}
```

Comment:
The condition (line 120) is wrong. You're setting `nodeSelector` if `grafana.updateStrategy` is set instead of `grafana.nodeSelector`

Reply:
Oh right - I just copy-pasted the existing one and changed the values. Forgot about the if clause.